src/index.js

```diff
@@ -80,6 +98,14 @@ export function withEffects(app) {
     }
 
     props.actions = enhanceActions(props.actions)
+    if (props.view) {
+      var originalView = props.view
+      props.view = function(state, actions) {
+        var nextVdom = originalView.apply(null, arguments)
```

Comment:
@okwolf .apply is slow, but if you call the original view with only the state and actions then you risk breaking other libraries that may be enhancing the function signature.

Having said that, with this kind of precaution means we wouldn't be able to do anything at all.

It's up to you, but if I was you, I would call `originalView(state, actions)`.

Reply:
I want to support variable number of modified `view` arguments. This is actually pretty close to what Babel spits out for:
```js
props.view = (...args) => {
  const nextVdom = originalView(...args)
  patchVdomEffects(args[1], nextVdom)
  return nextVdom
}
```

How much slower is `.apply()` exactly? [This jsPerf](https://jsperf.com/function-calls-direct-vs-apply-vs-call-vs-bind) implies that it's 5x slower than a direct call, which seems pretty crazy! 😮

Reply:
`...` is a language feature, so Babel has no better choice than `.apply` to implement spreads in ES5..

Reply:
@JorgeBucaran just how much of an anti-pattern would you say it is to augment the arguments to `view` in an HOA? 🤔

Reply:
@okwolf I think it will happen, isn't that what @zaceno is doing? By the way, I'd like to take the chance to suggest, instead of augmenting the view function, currying it like we do with actions! 😄

Reply:
@JorgeBucaran but then HOAs would still have to check if the original `view` returns a `function`, correct?

I'm going to actually go back to a direct call until someone complains about their special snowflake `view` ❄️

Reply:
Correct.